cmake/Modules/ObsHelpers_Windows.cmake

```diff
@@ -29,6 +29,9 @@ function(setup_binary_target target)
   endif()
 
   if(MSVC)
+    target_link_options(${target} PRIVATE
+                        /PDBALTPATH:$<TARGET_PDB_FILE_NAME:${target}>)
```

Comment:
I take it `PDBALTPATH` is used to avoid having that info written into the binary itself?

Reply:
By default MSVC writes the absolute path of the PDB file into the binary. This overrides that to be just the name (i.e. relative) so that binaries build in different environments remain identical.